docs/include/clone.rst

```diff
@@ -37,4 +37,5 @@
      --create-slot                 Create the replication slot
      --origin                      Use this Postgres replication origin node name
      --endpos                      Stop replaying changes when reaching this LSN
+     --disable-split-by-ctid	   	 Disable splitting tables by ctid
```

Comment:
Please keep the indentation consistent.

```suggestion
     --endpos                      Stop replaying changes when reaching this LSN
     --disable-split-by-ctid       Disable splitting tables by ctid
```

Reply:
I ran the linter, I wonder how it did not fix this

Reply:
EditorConfig is not a great linter. It is better than having none, I suppose.